packages/browser/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -98,9 +99,13 @@ export default class AuthCodeRedirectHandler implements IRedirectHandler {
     redirectUrl: string
   ): Promise<ISessionInfo & { fetch: typeof fetch }> {
     if (!(await this.canHandle(redirectUrl))) {
-      throw new ConfigurationError(
-        `Cannot handle redirect url [${redirectUrl}]`
-      );
+      // If the received IRI does not have redirection information, we can only
+      // return an unauthenticated session.
+      return {
```

Comment:
Should the "logged out session" be a constant somewhere?

Reply:
I initially meant to have it as a constant, but it was so small that I figured it wasn't necessary. If you feel like it could be useful, I'll change it.

Reply:
Fixed in e21486131070bc33b3f09d7a7f55e3030dda9771. While I was at it, I made sure the implicit flow behaves consistently.